### DISCUSSION TITLE
Minimum amount of points to issue a FEW

### DISCUSSION THREAD

**Discussion by regDaniel** [MeteoSwiss/ampycloud#112] [Q&A] [answered]
When looking at the changes introduced with #86 it is unclear to me if we would support setting MAX_HITS_OKTA0 = 0 and what would subsequently happen if we got just a single ceilo hit. Related:
https://github.com/MeteoSwiss/ampycloud/pull/86/files#r1348384355

**Comment by fpavogt** [comment, marked as answer]
Well, if one sets MAX_HITS_OKTA0 to 0, then layers with a single points will trigger a FEW code. Strictly speaking this is absolutely correct. However, it really means that one can trust every single ceilometer hit. I do not know the impact that this would have on a large scale, but in terms of the detailed "scientific stability" examples, it would only affect the case B.6 (from the article) by adding a layer. All other cases, the additional FEW layer is high up and would not be reported.

**Reply by regDaniel** [reply]
Thanks for your answer! This was not a scientific question at all, I was just wondering if the code acts predictable, if the user goes for this decision (be it reasonable or not) and it does 🙂 so all good!
